at_secondary/at_persistence_secondary_server/lib/src/notification/at_notification.dart

```diff
@@ -459,4 +460,26 @@ class AtNotificationBuilder {
     }
     return AtNotification._builder(this);
   }
+
+  reset() {
```

Comment:
Should we introduce reset along with the call? I am fine if there is a reason for this,

Reply:
The caller of the reset method in the at_secondary_server. Wanted to publish at_persistence_secondary_server and then use the reset method in the at_secondary_server